stats/utils.py

```diff
@@ -1,12 +1,19 @@
 import os
 import csv
 import logging
+
+from qgis.core import (
```

Comment:
Beter imports op losse regels. Dan doen we het allemaal (Lars, jij en ik) op dezelfde manier.